ts/modules/treemap.src.ts

```diff
@@ -1188,71 +1188,40 @@ seriesType<Highcharts.TreemapSeries>(
             });
         },
         setPointValues: function (this: Highcharts.TreemapSeries): void {
-            var series = this,
-                xAxis = series.xAxis,
-                yAxis = series.yAxis;
+            const series = this;
+            const { points, xAxis, yAxis } = series;
+            const styledMode = series.chart.styledMode;
 
-            series.points.forEach(function (
-                point: Highcharts.TreemapPoint
-            ): void {
-                var node = point.node,
-                    values = node.pointValues,
-                    x1,
-                    x2,
-                    y1,
-                    y2,
-                    crispCorr = 0;
-
-                // Get the crisp correction in classic mode. For this to work in
-                // styled mode, we would need to first add the shape (without x,
-                // y, width and height), then read the rendered stroke width
-                // using point.graphic.strokeWidth(), then modify and apply the
-                // shapeArgs. This applies also to column series, but the
-                // downside is performance and code complexity.
-                if (!series.chart.styledMode) {
-                    crispCorr = (
-                        (series.pointAttribs(point)['stroke-width'] || 0) % 2
-                    ) / 2;
-                }
+            const translateX = (value: number): number => Math.round(
+                xAxis.translate(value, false, false, false, true) as any
+            );
+            const translateY = (value: number): number => Math.round(
+                // Flip y-values so that 0 is top of the axis
+                yAxis.translate(100 - value, false, true, false, true) as any
```

Comment:
Won't be better to use `xAxis.toPixels(value, true)` and `yAxis.toPixels(100 - value, true)` instead of defining another `translateX(value)` and `translateY(value)` method?

Reply:
Thanks, good catch. `Axis.toPixels` is a better choice.
Was a bit unsure if I still should keep the local methods to avoid repeating rounding and the y-values logic, but I ended up removing the functions altogether.

Reply:
Thanks, good catch. Axis.toPixels is a better choice.
Was a bit unsure if I still should keep the local methods to avoid repeating rounding and the y-values logic, but I ended up removing the functions altogether.